cover-letter-pnas.md

```diff
@@ -0,0 +1,52 @@
+Dear Prof. Lenski,
+
+We would like to bring to your attention the manuscript entitled
+
+    Sustainable computational science: the ReScience initiative
+
+and ask you to consider it for publication in PNAS.
+
+The article documents the first two years of ReScience, an online
```

Comment:
The article documents the motivation for the initial creation and the first two...